python/py-path/Portfile

```diff
@@ -4,8 +4,8 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-path
-python.rootname     path.py
```

Comment:
I’ll change it back, but I don’t believe that it’s correct, and know it will break the tests in several of these ports because of the `from path import Path` line. You can verify this with the current `py-path` Portfile and the command:
```bash
python3 -c ‘from path import Path’
```

Reply:
ah okay, upstream has renamed this port from ```path.py``` to ```path```, so yes I can see how that would result in import failures.... But then you should make other changes to reflect that.

Reply:
I am traveling so don't have a chance to take a close look. But I suspect one can then remove the ```python.rootname``` line and keep the default as set by the PG, and keep the ```distname``` as it was. It will require though something different for the py27 subport as the latest supported version there will still be called ```path.py```. Also, updating the ```homepage``` would be good and perhaps other fields if needed.

Reply:
Thanks done.